server/scripts/stats.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable */
+const { dbClient } = require('../db/db.js');
+const { collections } = require('../db/schema.js');
+const inquirer = require('inquirer');
+const dayjs = require('dayjs');
+const utc = require('dayjs/plugin/utc');
+require('dotenv').config({ path: '../.env' });
```

Comment:
Is environment selection toggled via the .env? (e.g. dev/test/prod)

Reply:
The env selection is defined when you create the tunnel only.